### PR TITLE
Docs: add Cloud what's new items to 10.2 what's new

### PR DESCRIPTION
**Pull request by imatwawana**
Adds relevant items from _What's new in Grafana Cloud_ to _What's new in Grafana 10.2_ draft PR

Note for reviewers: This PR is just intended to confirm that you want your item added to the 10.2 What's new. If you don't, please let me know.

### CODE REVIEW

**Comment by imatwawana** [comment]
@alexweav - I've tagged you re: the **Grafana OnCall integration for Alerting** item since Brenda is away.

**Review by shelldandy — APPROVED** [review]
I see some todos in there but everything else specially for data sources lgtm

File: docs/sources/whatsnew/whats-new-in-v10-2.md [hunk 1]
```diff
@@ -201,6 +157,80 @@ TBD / WIP - some high level thoughts
...
+
+<!-- Joey Tawadrous, Jen Villa -->
+
+_Experimental in Grafana Open Source and Enterprise_
+
+(Requires Tempo or Grafana Enterprise Traces (GET) v2.2 or greater)
+
+We've added an **Aggregate By** option to the [TraceQL query editor](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/datasources/tempo/query-editor/traceql-search/#write-traceql-queries-using-search) to leverage Grafana Cloud Traces' [metrics summary API](https://grafana.com/docs/grafana-cloud/monitor-infrastructure/traces/metrics-summary-api/). You can calculate RED metrics (total span count, percent erroring spans, and latency information) for spans of `kind=server` received in the last hour that match your filter criteria, grouped by whatever attributes you specify.
+
+This feature is disabled by default. To enable it, file contact Grafana Support.
```
Comment (09jvilla) [inline docs/sources/whatsnew/whats-new-in-v10-2.md:172]:
```suggestion
This feature is disabled by default. To enable it, use the `metricsSummary` [experimental feature toggle](https://grafana.com/docs/grafana/next/setup-grafana/configure-grafana/feature-toggles/#experimental-feature-toggles).
```

File: docs/sources/whatsnew/whats-new-in-v10-2.md [hunk 2]
```diff
@@ -201,6 +157,80 @@ TBD / WIP - some high level thoughts
...
+
+### Tempo data source: "Aggregate By" Search option to compute RED metrics over spans aggregated by attribute
+
+<!-- Joey Tawadrous, Jen Villa -->
+
+_Experimental in Grafana Open Source and Enterprise_
+
+(Requires Tempo or Grafana Enterprise Traces (GET) v2.2 or greater)
+
+We've added an **Aggregate By** option to the [TraceQL query editor](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/datasources/tempo/query-editor/traceql-search/#write-traceql-queries-using-search) to leverage Grafana Cloud Traces' [metrics summary API](https://grafana.com/docs/grafana-cloud/monitor-infrastructure/traces/metrics-summary-api/). You can calculate RED metrics (total span count, percent erroring spans, and latency information) for spans of `kind=server` received in the last hour that match your filter criteria, grouped by whatever attributes you specify.
```
Comment (09jvilla) [inline docs/sources/whatsnew/whats-new-in-v10-2.md:170]:
```suggestion
We've added an **Aggregate By** option to the [TraceQL query editor](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/datasources/tempo/query-editor/traceql-search/#write-traceql-queries-using-search) to leverage Tempo's [metrics summary API](https://grafana.com/docs/tempo/latest/api_docs/metrics-summary/). You can calculate RED metrics (total span count, percent erroring spans, and latency information) for spans of `kind=server` received in the last hour that match your filter criteria, grouped by whatever attributes you specify.
```

**Review by 09jvilla — CHANGES_REQUESTED** [review]
Happy to have "Tempo data source: "Aggregate By" Search option to compute RED metrics over spans aggregated by attribute" in the What's New, but I suggested a few changes to make sure that the content was more open source in focus. Please check my links though - I just copy pasted the public URLs rather than trying to use relrefs.

**Comment by imatwawana** [comment]
> Happy to have "Tempo data source: "Aggregate By" Search option to compute RED metrics over spans aggregated by attribute" in the What's New, but I suggested a few changes to make sure that the content was more open source in focus. Please check my links though - I just copy pasted the public URLs rather than trying to use relrefs.

Those updates look great. Committed!

File: docs/sources/whatsnew/whats-new-in-v10-2.md [hunk 3]
```diff
@@ -201,6 +157,80 @@ TBD / WIP - some high level thoughts
...
+
+<!-- Joey Tawadrous, Jen Villa -->
+
+_Experimental in Grafana Open Source and Enterprise_
+
+(Requires Tempo or Grafana Enterprise Traces (GET) v2.2 or greater)
+
+We've added an **Aggregate By** option to the [TraceQL query editor](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/datasources/tempo/query-editor/traceql-search/#write-traceql-queries-using-search) to leverage Tempo's [metrics summary API](https://grafana.com/docs/tempo/<TEMPO_VERSION>/api_docs/metrics-summary/). You can calculate RED metrics (total span count, percent erroring spans, and latency information) for spans of `kind=server` received in the last hour that match your filter criteria, grouped by whatever attributes you specify.
+
+This feature is disabled by default. To enable it, use the `metricsSummary` [experimental feature toggle](https://grafana.com/docs/grafana/next/setup-grafana/configure-grafana/feature-toggles/#experimental-feature-toggles).
```
Comment (09jvilla) [inline docs/sources/whatsnew/whats-new-in-v10-2.md:172]: @imatwawana -- note that my link has a "next" in there. I'm not sure that's correct.

**Review by 09jvilla — APPROVED** [review]
Approving conditional on this one comment being addressed:
https://github.com/grafana/grafana/pull/76642/files#r1361154597

**Review by fabrizio-grafana — APPROVED** [review]

**Review by ktw4071 — APPROVED** [review]
Looks good for Datadog. Thank you!

**Review by alexweav — APPROVED** [review]

**Review by kylebrandt — APPROVED** [review]

File: docs/sources/whatsnew/whats-new-in-v10-2.md [hunk 4]
```diff
@@ -134,22 +95,19 @@ In this example, using the same data, with the min and max calculated for each i
 
 This is not only useful in the stat visualization; gauge, bar gauge, and status history visualizations, table cells formatted by thresholds, and gauge table cells all benefit from this addition.
 
-### Generative AI features for dashboards
+### New browse dashboards
```
Comment (joshhunt) [inline docs/sources/whatsnew/whats-new-in-v10-2.md:98]: Copied across the latest changes for new Browse Dashboards https://github.com/grafana/grafana/pull/76567 👍🏻

**Review by joshhunt — APPROVED** [review]
🚀

File: docs/sources/whatsnew/whats-new-in-v10-2.md [hunk 5]
```diff
@@ -201,6 +159,80 @@ TBD / WIP - some high level thoughts
...
+
+## Explore
+
+### Content outline
+
+<!-- Thanos Karachalios -->
+
+_Generally Available in Grafana Open Source and Enterprise_
+
+Introducing Content Outline in Grafana **Explore**. We recognized the challenges of complex mixed queries, as well as, lengthy logs and traces results, leading to time-consuming navigation and the loss of context. Content outline is our first step towards seamless navigation from log lines to traces and back to queries ensuring quicker searches while preserving context. Experience efficient, contextual investigations with this update in Grafana Explore. To learn more, refer to the [Content outline documentation](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/explore/#content-outline), as well as the following video demo.
```
Comment (harisrozajac) [inline docs/sources/whatsnew/whats-new-in-v10-2.md:232]:
```suggestion
Introducing Content Outline in Grafana **Explore**. We recognized the challenges of complex mixed queries, lengthy logs and traces results, leading to time-consuming navigation and the loss of context. Content outline is our first step towards seamless navigation from log lines to traces and back to queries ensuring quicker searches while preserving context. Experience efficient, contextual investigations with this update in Grafana Explore. To learn more, refer to the [Content outline documentation](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/explore/#content-outline), as well as the following video demo.
```

File: docs/sources/whatsnew/whats-new-in-v10-2.md [hunk 6]
```diff
@@ -40,81 +40,42 @@ Use full URLs for links. When linking to versioned docs, replace the version wit
...
+## Public dashboards
 
-With the current release, we've introduced a new configuration option for each OAuth provider called `use_refresh_token` that allows you to configure whether the particular OAuth integration should use refresh tokens to automatically refresh access tokens when they expire. In addition, to further improve security and provide secure defaults, `use_refresh_token` is enabled by default for providers that support either refreshing tokens automatically or client-controlled fetching of refresh tokens. It's enabled by default for the following OAuth providers: `AzureAD`, `GitLab`, `Google`.
+<!-- Thanos Karachalios -->
 
-For more information on how to set up refresh token handling, please refer to [the documentation of the particular OAuth provider.](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/setup-grafana/configure-security/configure-authentication/).
+_Generally Available in Grafana Enterprise_
```
Comment (evictorero) [inline docs/sources/whatsnew/whats-new-in-v10-2.md:47]: Not only on Enterprise, this will be GA in OSS as well.

Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-in-v10-2.md:47]:
```suggestion
_Generally Available in Grafana Open Source and Enterprise_
```

File: docs/sources/whatsnew/whats-new-in-v10-2.md [hunk 7]
```diff
@@ -201,6 +159,80 @@ TBD / WIP - some high level thoughts
...
+
+## Explore
+
+### Content outline
+
+<!-- Thanos Karachalios -->
+
+_Generally Available in Grafana Open Source and Enterprise_
+
+Introducing Content Outline in Grafana **Explore**. We recognized that complex mixed queries, as well was lengthy logs and traces results led to to time-consuming navigation and the loss of context. Content outline is our first step towards seamless navigation from log lines to traces and back to queries ensuring quicker searches while preserving context. Experience efficient, contextual investigations with this update in Grafana Explore. To learn more, refer to the [Content outline documentation](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/explore/#content-outline), as well as the following video demo.
```
Comment (harisrozajac) [inline docs/sources/whatsnew/whats-new-in-v10-2.md:232]:
```suggestion
Introducing Content Outline in Grafana **Explore**. We recognized that complex mixed queries, as well as lengthy logs and traces results led to to time-consuming navigation and the loss of context. Content outline is our first step towards seamless navigation from log lines to traces and back to queries ensuring quicker searches while preserving context. Experience efficient, contextual investigations with this update in Grafana Explore. To learn more, refer to the [Content outline documentation](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/explore/#content-outline), as well as the following video demo.
```

Comment (imatwawana) [reply docs/sources/whatsnew/whats-new-in-v10-2.md:232]: Thanks for the catch!

**Pull request merged**